src/index.ts

```diff
@@ -24,33 +23,41 @@ type GetUser = {
   email: string;
 };
 
+type $TSFixMe = any;
```

Comment:
It is the proper type here.

If you look into previous code it used to send type `any`, and
`JWTPayload` returns data of type `Record<"some-key-1"|"some-key-2"|...|string, string | unknown` which gives ts-warning of unknown types as expected.

Using the convention `$TSFixMe` is used to declare type for data who's exact type is unknown because of limitation/some-reason and will be resolved in future.

For More Why I used the type please check this video(time appended): https://youtu.be/y7WUsi6NeH8?t=532

Reply:
You can leave this part for now. We might need to create a new type for this in the future.

Reply:
Better and good alternative: `Record<string, $TSFixME>`